DigitalLearningSolutions.Data/Helpers/CentreEmailHelper.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace DigitalLearningSolutions.Data.Helpers
+{
+    public static class CentreEmailHelper
+    {
+        public static string GetEmailForCentreNotifications(

```

Comment:
Could do with a very basic unit test for this.

Additionally, as the scope has changed, we are going to want the ticket description changed to match the new requirements.